internal/commands/vaultsecrets/secrets/displayer.go

```diff
@@ -50,6 +53,15 @@ func (d *displayer) SetDefaultFormat(f format.Format) *displayer {
 	return d
 }
 
+func (d *displayer) SetSingleSecret(s string) *displayer {
+	d.secretType = s
+	return d
+}
+
+func (d *displayer) single() bool {
+	return d.secretType != ""
+}
```

Comment:
!= "kv"? It feels like secretType should never be empty.

Reply:
Was able to remove this entirely.